lib/nerves_hub_link/update_manager.ex

```diff
@@ -61,6 +67,14 @@ defmodule NervesHubLink.UpdateManager do
     GenServer.call(manager, :status)
   end
 
+  @doc """
+  Returns the current status of the update manager
```

Comment:
```suggestion
  Returns the previous update status of the update manager
```

This says current status, but the function is previous update?

Reply:
Thanks for spotting this!